src/functions/sendslack/handler.ts

```diff
@@ -0,0 +1,130 @@
+import type { ValidatedEventAPIGatewayProxyEvent } from "@libs/api-gateway";
+import { formatJSONResponse } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+require("dotenv").config({ path: __dirname + "/../../../../../.env" });
+import { WebClient, LogLevel } from "@slack/web-api";
+import moment from 'moment';
+
+import { PersonDto, TimeEntryTotalDto, UseTimebankData } from "./schema";
+import timebankData from "./timebankdata";
+import schema from "./schema";
+
+/**
+ * handler function for all Api calls/ message posts when AWS scheduled
+ * 
+ * @param event 
+ * @returns JSON response
+ */
+const sendSlack: ValidatedEventAPIGatewayProxyEvent<typeof schema> = async (event) => {
+  const {
+    personData,
+    timeData,
+  }: { personData: PersonDto[]; timeData: TimeEntryTotalDto[] } = await timebankData();
+
+  const formatPersonData = personData.map((person) => {
+    return { id: person.id, name: person.first_name + " " + person.last_name };
+  });
+
+  // Formatting of the timebankData
+  const useTimebankData: UseTimebankData[] = timeData.map((person, i) => {
+    return {
+      ...formatPersonData[i],
```

Comment:
I wouldn't trust that the lists are always in same order. Also I would combine these functionalities into single function and move it outside this file to some utility file.